### DISCUSSION TITLE
/cn/2020/06/bayesian-optimization/

### DISCUSSION THREAD

**Discussion by leovan** [leovan/leovan.me#15] [Comments]
贝叶斯优化 (Bayesian Optimization) 
 https://leovan.me/cn/2020/06/bayesian-optimization/

**Comment by leovan** [comment]
```header
由 1tz 于 2021-08-19 09:58:03 写道：
````
写得挺好👍

**Reply by leovan** [reply]
```header
于 2021-11-22 01:26:13 写道：
````
谢谢。

**Comment by leovan** [comment]
```header
由 徐晓芬 于 2021-11-10 08:22:01 写道：
````
哈哈哈留个脚印\~找贝叶斯优化相关意外发现了一个有趣的大佬，想问下GP优化能否应用于输入变量相关非独立的场景呢？

**Reply by leovan** [reply]
```header
于 2021-11-22 01:30:02 写道：
````
首先需要承认我对这个领域也不是很熟悉，当时总结这边也是从入门科普的角度。个人的一点看法吧，高斯过程背后有着强大的数学理论支持，从严谨的角度，对于参数和观测值都有一些要求。但现实生活中应该也不存在完全不相关的变量，感觉可以作实验看看。例如找一个有相关输入变量的问题，一个直接用高斯过程尝试一下，另外一个先去一下相关性（例如：PCA），再用高斯过程试一试，对比看看效果如何？

**Reply by leovan** [reply]
```header
由 徐晓芬 于 2021-11-29 03:05:23 写道：
````
有道理！实验出真知

**Comment by leovan** [comment]
```header
由 Guo Liang Zhang 于 2021-12-17 02:05:31 写道：
````
意外遇见一位大佬\~

**Comment by leovan** [comment]
```header
由 jiang13021 于 2021-12-21 09:31:00 写道：
````
写的不错👍，但采集函数部分，最好应该指出\\Phi 和 \\phi
两个函数是标准正态分布的累积分布函数和概率密度函数。而且第27个式子，是不是多打了个右括号？

**Reply by leovan** [reply]
```header
于 2021-12-28 13:19:03 写道：
````
感谢，建议已经采纳。不过再三检查了下，第 27
个式子应该是没问题的，没多右括号。

**Comment by luckydognotbad** [comment]
哥，你真的太帅了，帅炸了

**Comment by HaibinLai** [comment]
大佬！想问问你的贝叶斯优化动态演示是怎么实现的！太厉害了！

**Reply by leovan** [reply]
是将多个图片做了个简单的前端播放器，实现的 Hugo Shortcode 可参考：<https://github.com/leovan/leovan.me/blob/main/layouts/shortcodes/figure-slider.html>，当然这也是参考的前人之作：<https://distill.pub/2020/bayesian-optimization/>。

**Comment by siyuan1979** [comment]
写得挺不错的，问个问题，PI和EI采集函数都给出了公式，并没有给出用于迭代的下一个点的值的实际算法，如果采用一阶导数等于0的算法，可能存在多个最大值(或最小值点)，需要一个一个的比较选择？有没有其他的方法；另外，贝叶斯优化的效率如何？（和ga，sa，pso，aco等其他算法）
